services/extra_classes.txt

```diff
@@ -0,0 +1 @@
+services_autorun
```

Comment:
Because it's really convenient to say `echo services_autorun >> extra_classes.txt` :)

I commented it out but left the file in place, so it's empty and ready to use.